chemprop/sklearn_predict.py

```diff
@@ -19,13 +19,8 @@ def predict_sklearn(args: SklearnPredictArgs) -> None:
                  loading data, loading a trained scikit-learn model, and making predictions with the model.
     """
     print('Loading data')
-    data = get_data(
-        path=args.test_path,
-        smiles_column=args.smiles_column,
-        target_columns=[],
-        ignore_columns=[],
-        store_row=True
-    )
+    data = get_data(path=args.test_path, smiles_column=args.smiles_column, target_columns=[], ignore_columns=[],
```

Comment:
Two comments which also apply to some later changes as well:

1) Is there a need to add `args=args`, especially if you remove the dependence on `number_of_molecules`? Although I haven't done a great job of it, I think it's better to avoid passing around `args` where possible and instead to make the parameters of functions explicit.

2) When there are a lot of parameters to a function, I generally like to put them all on separate lines to make it a little easier to read, even if it takes up more lines. Could you do that too?